dask/bag/core.py

```diff
@@ -1101,26 +1101,32 @@ def from_castra(x, columns=None, index=False, npartitions=None):
         x = Castra(x.path, readonly=True)
     if columns is None:
         columns = x.columns
-    if npartitions is None:
```

Comment:
Ping @mrocklin: Just saw this when referenced from another issue. This removed support for `npartitions` (including the tests), but didn't remove the docs or keyword from the function. I added this keyword to allow bags with fewer partitions than the castra has partitions (specifically I wanted same number of partitions as I had cores, used in [the reddit blog post here](http://blaze.pydata.org/blog/2015/09/08/reddit-comments/#What-about-Non-DataFrame-Computations?)). It did complicate things though, but also mildly improved performance. Could either add the support back, or remove the keyword/docs. Up to you.